### ISSUE TITLE
How can I incorporate textual inversion into the UniControl? Will unicontrol support textual inversion?

### ISSUE THREAD

**Issue by Howard-Hsiao** [salesforce/UniControl#22]
Hello, I'm aiming to incorporate textual inversion into your impressive project. To start, my initial objective is to extract the stable diffusion weights from the unicontrol_v1.1.ckpt model and then load these weights to the Hugging Face diffusers pipeline for training textual inversion. Subsequently, I'll need to introduce the requisite code to enable the model to employ the trained embedding.

Does this approach seem viable? If it does, could you guide me on the precise steps to successfully extract the stable diffusion weights from the unicontrol model?

**Comment by canqin001** [comment]
Yes. I believe that stable diffusion can be extracted from unicontrol_v1.1.ckpt. Please feel free to let me know anything I can help.
